convertToSingleHost.js

```diff
@@ -40,38 +40,15 @@ async function convertProjectToSingleHost(host) {
   await writeFileAsync(`./manifest.xml`, manifestContent);
 
   // copy over host-specific taskpane code to taskpane.ts
-  const srcContent = await readFileAsync(`./src/taskpane/app/${host}.app.component.ts`, "utf8");
-  await writeFileAsync(`./src/taskpane/app/app.component.ts`, srcContent);
-
-  // delete all test files by default for now - eventually we want to convert the tests by default
-  if (convertTest && (host === "excel" || host === "word")) {
```

Comment:
Why are we removing all this?  I believe it's not being used at the moment . . . but if we did remove this then it should be something that is done across the board and not just for one template.

Reply:
Aggree, we can keep it and, if needed, remove it in the taskpane repo